WordPress/Classes/Services/BlogJetpackSettingsService.swift

```diff
@@ -0,0 +1,194 @@
+                                    key: BlogJetpackSettingsServiceRemote.Keys.blockMaliciousLoginAttemptsKey,
+                                    value: value as AnyObject,
+                                    success: success,
+                                    failure: failure)
+    }
+
+    func updateWhiteListedIPAddressesForBlog(_ blog: Blog, value: Set<String>, success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
+        let joinedIPs = value.joined(separator: ", ")
+        updateJetpackSettingForBlog(blog,
+                                    key: BlogJetpackSettingsServiceRemote.Keys.whiteListedIPAddressesKey,
+                                    value: joinedIPs as AnyObject,
+                                    success: success,
+                                    failure: failure)
+    }
+
+    func updateSSOEnabledForBlog(_ blog: Blog, value: Bool, success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
+        updateJetpackSettingForBlog(blog,
+                                    key: BlogJetpackSettingsServiceRemote.Keys.ssoEnabledKey,
+                                    value: value as AnyObject,
+                                    success: success,
+                                    failure: failure)
+    }
+
+    func updateSSOMatchAccountsByEmailForBlog(_ blog: Blog, value: Bool, success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
+        updateJetpackSettingForBlog(blog,
+                                    key: BlogJetpackSettingsServiceRemote.Keys.ssoMatchAccountsByEmailKey,
+                                    value: value as AnyObject,
+                                    success: success,
+                                    failure: failure)
+    }
+
+    func updateSSORequireTwoStepAuthenticationForBlog(_ blog: Blog, value: Bool, success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
+        updateJetpackSettingForBlog(blog,
+                                    key: BlogJetpackSettingsServiceRemote.Keys.ssoRequireTwoStepAuthenticationKey,
+                                    value: value as AnyObject,
+                                    success: success,
+                                    failure: failure)
+    }
+
+    func updateJetpackMonitorSettinsForBlog(_ blog: Blog, success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
+        guard let blogInContext = self.context.object(with: blog.objectID) as? Blog,
+            let remote = BlogJetpackSettingsServiceRemote(wordPressComRestApi: blog.wordPressComRestApi()),
+            let blogDotComId = blogInContext.dotComID as? Int else {
+                failure(nil)
+                return
+        }
+
+        remote.updateJetpackMonitorSettingsForSite(blogDotComId,
+                                                   settings: self.jetpackMonitorsSettingsRemote(blogInContext.settings!),
+                                                   success: { () in
```

Comment:
I think I haven't seen the `() in` syntax/convention before. If there are no parameters it can be left blank, but I wonder if that's there for a reason. Not really a request to change it, just curious.

Reply:
Force of habit, removed now.